src/DataViews/SamplingReportDataView.cpp

```diff
@@ -340,6 +339,34 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
   stack_events_count_ = stack_events_count;
 }
 
+std::string SamplingReportDataView::GetToolTip(int row, int column) {
+  if (column != kColumnInclusive && column != kColumnExclusive) {
+    return "";
+  }
+  const SampledFunction& function = GetSampledFunction(row);
+  uint32_t raw_count = {};
+  float percentage = {};
+  if (column == kColumnInclusive) {
+    raw_count = function.inclusive;
+    percentage = function.inclusive_percent;
+  } else {
+    raw_count = function.exclusive;
+    percentage = function.exclusive_percent;
+  }
+
+  orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(percentage / 100.0f, stack_events_count_);
+
+  return absl::StrFormat(
```

Comment:
I think that this tooltip is a bit problematic because it's identical for the "inclusive" and "exclusive" case, and it's not super easy to realize that it depends on what column you are hovering on.
As a matter of fact, for the exclusive case "encountered %u times in a total of %u stack samples" is incorrect, because we mean the number of times in which the function was _at the top of the stack_.
So I think we should:
- phrase the two cases differently;
- maybe find a way to add the words "inclusive" and "exclusive" in the two different cases.

For example:
"The function %s was encountered %u times (inclusive count) in a total of %u stack samples."
vs
"The function %s was at the top of the callstack %u times (exclusive count) in a total of %u stack samples." / "The function %s was the innermost function %s times (exclusive count) in a total of %u stack samples".

What do you think?

Reply:
I thought, it's clear enough, as the user knows which cell is being hovered over. But if smb thinks, smth might go wrong, it probably will. 

How about we write 

"The function %s was encountered %u times on _top of the callstack_ in a total of %u stack samples" vs 
"The function %s was encountered %u times on _anywhere on the stack_ in a total of %u stack samples" ?

Reply:
OTOH, we assume the terms inclusive/exclusive are clear enough. Why do you suggest to explain what "exclusive count" means with "at the top of the callstack"/"the innermost function" ?

Reply:
> I thought, it's clear enough, as the user knows which cell is being hovered over.

Not really. For example, in other cases you could have a single tooltip for the whole row. Having almost identical tooltips is confusing.

> we assume the terms inclusive/exclusive are clear enough

We don't, we more or less all agree this can be explained better, and this is the perfect moment to add it.

> Why do you suggest to explain what "exclusive count" means with "at the top of the callstack"/"the innermost function"

"encountered %u times in a total of %u stack samples" is wrong for exclusive, so this is first and foremost about making it correct.

> "The function %s was encountered %u times on top of the callstack in a total of %u stack samples" vs
> "The function %s was encountered %u times on anywhere on the stack in a total of %u stack samples" ?

on top of the callstack -> at the top of the callstack (in the top position, not above the callstack)
on anywhere on the stack -> anywhere in the callstack

I don't like the "anywhere" phrasing too much, the sentence sounds a bit weird overall. Not sure how to explain it, I guess it's fine. It can also be removed as per my original suggestion.